dotCMS/src/main/webapp/WEB-INF/velocity/static/content/content_form_macro.vtl

```diff
@@ -1,1110 +1,911 @@
-								document.getElementById("alert"+"${field.velocityVarName}").innerHTML = customErrorMessages["$field.velocityVarName"];
-							else
-								document.getElementById("alert"+"${field.velocityVarName}").innerHTML = ${field.velocityVarName}Empty;
-
-							return false;
-						}
-					#end
-				#end
-
-
-				#if($UtilMethods.isSet($field.getRegexCheck()))
-
-					#set($customErrorMessage = $secureformtool.getFieldVariableValue($structure.getVelocityVarName(), $field.velocityVarName, "errorFieldMessage"))
-					#if($UtilMethods.isSet($customErrorMessage))
-						customErrorMessages["$field.velocityVarName"]="$customErrorMessage";
-						#set($customErrorMessage ="")
-					#else
-						customErrorMessages["$field.velocityVarName"]="";
-					#end
-
-
-					#set($regExp = $field.getRegexCheck())
-					if("$!regExp" != ""){
-						#if($regExp.equals("[^(<[.\n]+>)]*"))
-							#set($regExp = $regExp.replaceAll("\(","").replaceAll("\)",""))
-							if(!validateRegularExpresionNotHTML(document.getElementById("$field.velocityVarName"), "$UtilMethods.makeJavaSafe($regExp)", "$field.fieldName")){
-							return false;
-						}	
-						#else
-			       		if(!validateRegularExpresion(document.getElementById("$field.velocityVarName"), "$UtilMethods.makeJavaSafe($regExp)", "$field.fieldName")){
-							return false;
-						}
-
-						#end	
-					}
-					
-				#end
-			#end
-    	#end
-
-    	#if($contentUseCaptcha || $contentUseAudioCaptcha)
-    		 customErrorMessages["captcha"]="";
-	    	if(!lengthValid(document.getElementById("captcha"), 1, "captcha")){
-				return false;
-			}
-	    #end
-    }
-
-
-    return true;
-}
-
-var captcha_audio_play;
-
-	function playAudioCaptcha(){
-		captcha_audio_play = document.getElementById('captcha-audio-play');
-		captcha_audio_play.setAttribute("class","captcha-audio-playing");
-		var rand=Math.floor(Math.random()*999999999);
-		if(dojo.isIE){
-			document.getElementById('captcha-audio').innerHTML = '<embed hidden="false" src="/audioCaptcha.wav?r=' + rand +'" id="captcha-sound" controller="false" width="0" height="0" autoplay="true" autostart="true" type="audio/wav" />';
-		}else{
-			document.getElementById('captcha-audio').innerHTML = '<audio hidden="false" src="/audioCaptcha.wav?r=' + rand +'" id="captcha-sound" controller="false" width="0" height="0" autoplay="true" autostart="true" type="audio/wav" />';
-		}
-
-		setTimeout(function(){
-			captcha_audio_play.setAttribute("class","audio-play");
-		},8000);
-
-	}
-
-
-	function resetCaptcha(){
-		var rand=Math.floor(Math.random()*999999999)
-		document.getElementById('captcha-image').innerHTML = "<img width='200' height='50'  src='/Captcha.jpg?r=" + rand + "'>";
-	}
-
-    function userLang() {
-        return "$text.getFromSessionLanguage('org.dotcms.frontend.content.submission.does.not.comply.format')";
-	}
-
-updateTabList();
-
-
-
-
-
-
-
-
-
-</script>
+#set($dojoPath = $webapi.getConfigVar("path.to.dojo"))
```

Comment:
it seems the content was replaced or reformatted, just double check if this is ok